app/models/git/track.rb

```diff
@@ -19,14 +19,29 @@ def initialize(git_sha = "HEAD", repo_url: nil, repo: nil)
       @git_sha = git_sha
     end
 
-    def test_regexp
-      pattern = config[:test_pattern]
-      Regexp.new(pattern.presence || "[tT]est")
+    memoize
+    def title
+      config[:language]
+    end
+
+    memoize
+    def slug
+      config[:slug]
     end
 
-    def ignore_regexp
-      pattern = config[:ignore_pattern]
-      Regexp.new(pattern.presence || "[iI]gnore")
+    memoize
+    def blurb
+      config[:blurb]
+    end
+
+    memoize
+    def tags
+      config[:tags].to_a
+    end
+
+    memoize
+    def active?
+      config[:active].present?
```

Comment:
I have a small preference to `!!config[:active]` here. I wasn't immediately sure whether this handles `false` correctly (which it does). Also happy to stick with `present?` if you (or Rubocop) prefer it.

Reply:
Sure.